### PR TITLE
INT- B-19713-UI-Address-PPM-Shipment-Card

### PR DESCRIPTION
**Pull request by MInthavongsay**
## Summary

**INTEGRATION**

Simple PR to display Origin/Destination addresses on the PPM shipment card view on customer summary page view. Origin/Destination are labeled as "From/To" in the following format in short address format(city, state, postal):  
_From: Somewhere, AL 12345 To: Somewhere, AL 12345_ 

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. As customer create and submit move containing a PPM.
2. After submission verify From/To addresses are not displayed. It will be only be displayed when service counselor reviews/approves for pre-closeout and closeout. 
Screenshot 2024-06-03 at 2 37 33 PM
3. As new service counselor, review and approve for Pre-Closeout. Click <submit move details>
4. As customer, view summary page and verify "From/To" label is displayed for origin/destination addresses. **Note: old screenshot. The final product is "From/To". Not "To/From" shown in the screenshot.** 
Screenshot 2024-06-03 at 1 05 53 PM
5. Click <Upload PPM Documents> and upload docs and submit.
6. As counselor, review documents for PPM, APPROVE and Confirm. (Closeout)
7. As customer verify PPM shipment card view displays "From/To" with disabled <Upload PPM Documents>.
Screenshot 2024-06-03 at 2 38 03 PM

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### CODE REVIEW

File: src/components/PPMSummaryList/PPMSummaryList.jsx [hunk 1]
```diff
@@ -7,10 +7,23 @@ import styles from './PPMSummaryList.module.scss';
...
+      <p>
+        <span className={styles.bold}>From: </span>
+        {formatAddressShort(pickupAddress)}
+        <span className={styles.bold}> To: </span>
+        {formatAddressShort(destinationAddress)}
+      </p>
+    </div>
+  );
+};
+
```
Comment (CoryKleinjanCACI) [inline src/components/PPMSummaryList/PPMSummaryList.jsx:26]: I think this could look cleaner if the From/To were on seperate lines, but not a big deal.

**Review by CoryKleinjanCACI — APPROVED** [review]
Tests pass and code changes look good.

File: src/components/PPMSummaryList/PPMSummaryList.jsx (hunk 1 above)
Comment (MInthavongsay) [reply src/components/PPMSummaryList/PPMSummaryList.jsx:26]: Dan M wanted it in one line.

**Review by ajlusk — APPROVED** [review]
UI changes display correctly. Tests (which appear to be 75% of this ticket) pass locally.

Might make more sense to rename `toFromAddressDisplay()` to `fromToAddressDisplay()` due to reordering how they display but it's current name still gets the point across.

**Pull request merged**